packages/datagrid/src/components/DatasetSerializer/datasetSerializer.js

```diff
@@ -55,9 +56,26 @@ export function sanitizeAvro(avro) {
  * @param {boolean} optional;
  */
 export function getTypeValue(type, optional) {
+	if (typeof type === 'string') {
+		return `${type}${optional ? '' : '*'}`;
+	}
 	return `${type.dqType || type.type}${optional ? '' : '*'}`;
 }
 
+/**
+ * isNull - check if the type is null
```

Comment:
No need to repeat the function name
```suggestion
  * Check if the type is null
```